augraphy/base/paperfactory.py

```diff
@@ -65,6 +65,9 @@ def __call__(self, image, layer=None, force=False):
                 self.texture_file_names = []
                 self.paper_textures = []
 
+                # check for edge
+                texture = self.check_paper_edges(texture)
```

Comment:
With this update, it should remove most of the unnecessary border section of paper texture, for example:

left - original, right - cropped image
![image](https://user-images.githubusercontent.com/52521979/166147435-1c9dddd1-3160-475f-984e-457fb568e860.png)

![image](https://user-images.githubusercontent.com/52521979/166147446-b9a39e2f-ab05-4f05-90ac-738f2511b6f9.png)

![image](https://user-images.githubusercontent.com/52521979/166147452-886705d2-f640-45d1-bb56-daf9da4c6b03.png)

![image](https://user-images.githubusercontent.com/52521979/166147482-6b27b941-9cee-439d-b555-de9574dd0516.png)